README.md

```diff
@@ -1,2 +1,42 @@
-historic_cadastre
-=================
+# historic_cadastre
+
+## Installation
+
+First, get the code:
+
+    Drive:
+    cd where_ever\historic_cadastre
+    git init
+    git remote add upstream https://github.com/sitn/historic_cadastre.git
+    git fetch upstream
+    git merge upstream/master
+
+Next, run bootstrap (we are assuming that Python is in our path):
+
+    python bootstrap-buildout.py --allow-site-packages
```

Comment:
Yes... sorry... I did not re-read this file... This PR is quite old... I will refactor this file as well...